docusaurus.config.js

```diff
@@ -3,42 +3,47 @@ const darkCodeTheme = require('prism-react-renderer/themes/vsDark');
 
 /** @type {import('@docusaurus/types').DocusaurusConfig} */
 module.exports = {
-  title: 'IPC144 - Course Notes',
-  tagline: 'C Programming',
-  url: 'https://cghub.ca',
-  baseUrl: '/',
+  title: "IPC144 - Course Notes",
```

Comment:
Please don't touch every line in a file for style changes, it makes reviewing harder than it needs to be and messes with git history.

Reply:
my linter was has a rule set to use double quotes I believe which changed the file automatically, I will fix my settings